05.host/08.maintain-domain/docs.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Maintain Your Domain
+taxonomy:
+    category: docs
+
+---
+
+Maintain your High Fidelity domain by clearing it of clutter from recently created entities. For example, you can maintain your domain by deleting all entities created in the last hour. This allows you to keep your domain clean and in a good state. 
+
+**On This Page**
+
++ [Set Temporary Create Permissions](#set-temporary-create-permissions)
++ [Use an AC Script to Clean Your Domain](#use-an-ac-script-to-clean-your-domain)
+
+## Set Temporary Create Permissions
+
+You can [set different user permissions](../your-domain/secure-domain.html#set-user-permissions) for your domain. These permissions will determine the types of users that have the permission to edit your domain. When you set these permissions, ensure that you have locked your existing entities or set entity filters to prevent users from editing objects already present in your domain.
```

Comment:
The intro paragraph here needs to connect 'temporary permissions' to the topic of this page -- maintaining your domain. Its not immediately clear how this option "maintains" your domain.

Reply:
Done